### PR TITLE
Small priority two changes

### PR DESCRIPTION
**Pull request by gibsonbailey**
This PR includes three bullets from Barcelona: Scope Item 5 (Alpha Support & Bug Fixes). These have been grouped together because they are particularly small and simple tasks.

* Remove the uniqueness constraint on class numbers
* Place Codalab challenge button on the submit homework page
* Test-Your-Class button should point to the instructor’s automatically-created student profile’s student view

### CODE REVIEW

File: apps/klasses/models.py [hunk 1]
```diff
@@ -35,7 +35,7 @@ class Klass(models.Model):
     instructor = models.ForeignKey('profiles.Instructor', related_name='klasses', on_delete=models.CASCADE)
 
     title = models.CharField(max_length=60, null=False, blank=False, default="New Course")
-    course_number = models.SlugField(max_length=60, null=False, blank=False, unique=True, default=None)
+    course_number = models.SlugField(max_length=60, null=False, blank=False, unique=False, default=None)
```
Comment (ckcollab) [inline apps/klasses/models.py]: can remove this kwarg if using default?

File: apps/klasses/models.py [hunk 2]
```diff
@@ -17,14 +17,14 @@
 def upload_image(instance, filename):
     file_split = filename.split('.')
     file_extension = file_split[len(file_split) - 1]
-    path = "images/{0}/class_image.{1}".format(instance.course_number, file_extension)
+    path = "images/{0}/{1}/class_image.{2}".format(instance.pk, instance.course_number, file_extension)
```
Comment (ckcollab) [inline apps/klasses/models.py]: can we be using f strings on this project?

**Review by ckcollab — CHANGES_REQUESTED** [review]

File: apps/klasses/models.py (hunk 1 above)
Comment (ckcollab) [reply apps/klasses/models.py]: Also `null=False` but you are setting `default=None`?

**Review by ckcollab — CHANGES_REQUESTED** [review]

File: apps/klasses/models.py (hunk 1 above)
Comment (gibsonbailey) [reply apps/klasses/models.py]: Great point I'll remove the default setting.

File: apps/klasses/models.py (hunk 2 above)
Comment (gibsonbailey) [reply apps/klasses/models.py]: Yes! I should have caught this!

Comment (gibsonbailey) [reply apps/klasses/models.py]: ✅

**Pull request merged**